README.md

```diff
@@ -30,21 +30,29 @@ See the installation in [its own page](https://github.com/areee/dart_kassakuitti
 The basic usage looks like this:
```

Comment:
Add something like this to the beginning of the readme file:

> Works with macOS, Linux and Windows.

Of course you know certainly that it works at least with macOS and supposed also with Linux (because of Unix). But especially Windows needs testing before adding this line! :)

Reply:
Another suggestion to add: add some **sample data** (HTML files from S-kaupat and K-ruoka [= EAN products] and a text file from S-kaupat [= receipt products]). Of course they should be limited to not show all private details.

Reply:
> Another suggestion to add: add some sample data (HTML files from S-kaupat and K-ruoka [= EAN products] and a text file from S-kaupat [= receipt products]). Of course they should be limited to not show all private details.

I think this is too risky, especially the HTML files. I'll forget this idea.

Reply:
I've now tested the program with macOS, Linux and Windows. It seems to work correctly.